src/engine/enginebuffer.cpp

```diff
@@ -509,6 +511,7 @@ void EngineBuffer::slotTrackLoaded(TrackPointer pTrack,
     TrackPointer pOldTrack = m_pCurrentTrack;
 
     m_pause.lock();
+    m_pTrackLoaded->set(1);
```

Comment:
The user might think that all other track related COs are valid after TrackLoaded is 1.0. 
So setting the CO should be the very last action.